src/examples/color-swatch/ColorSwatchCustomTrigger.tsx

```diff
@@ -74,11 +74,11 @@ const Example = () => {
   const [selectedColIndex, setSelectedColIndex] = useState(SELECTED_COL_INDEX);
 
   const onSelect = (rowIdx: number | null, colIdx: number | null) => {
-    if (typeof rowIdx !== 'number' || typeof colIdx !== 'number') return;
```

Comment:
No one-line `if` statements. Need to get Garden's `eslint-config` updated accordingly.